assets/sass/06-components/navigation.scss

```diff
@@ -212,7 +212,7 @@
 		// Sub-menu buttons
 		.sub-menu-toggle {
 			display: flex;
-			height: 100%;
+			height: calc(2 * var(--primary-nav--padding) + 1.15em + 1px);
```

Comment:
I know this is already merged, but it would be nice to see a comment explaining where this formula came from.